models/pipeline.js

```diff
@@ -52,7 +52,15 @@ const MODEL = {
 
     lastEventId: Joi.number().integer().positive()
         .description('Identifier of last event')
-        .example(123345)
+        .example(123345),
+
+    configPipelineId: Joi.number().integer().positive()
+        .description('Identifier of pipeline containing external configuration')
+        .example(123),
+
+    isConfigPipeline: Joi.boolean()
```

Comment:
can use use presence of `scmUrls` to infer this ?

Reply:
yep, makes sense!

Reply:
Then need to add `scmUrls` to this pipeline model @